addons/event_sale/models/event.py

```diff
@@ -21,6 +21,7 @@ class event_event(models.Model):
     badge_back = fields.Html('Badge Back', translate=True, states={'done': [('readonly', True)]})
     badge_innerleft = fields.Html('Badge Innner Left', translate=True, states={'done': [('readonly', True)]})
     badge_innerright = fields.Html('Badge Inner Right', translate=True, states={'done': [('readonly', True)]})
+    is_unlimited = fields.Boolean('Unlimited Tickets', compute='_compute_seats_availability')
```

Comment:
Ok, thanks for pointing it out. But then I think your implementation does not match APR's comments. It was said that event seats limit and tickets seats limit are completely separated. You may have unlimited tickets for a limited event (i.e. I have 50 available sears, but any number of "basic", "silver" and "gold" subscriptions -> the event is still limited). Could you confirm this is the intended behavior ?